manifests/init.pp

```diff
@@ -119,9 +120,9 @@
 #   include hdm
 class hdm (
   # installation parameter
+  String[1]                     $version,
   Enum['docker', 'rvm']         $method                = 'docker',
   Boolean                       $manage_docker         = true,
-  String[1]                     $version               = 'main',
```

Comment:
can't we set `1.0.1` as default version? Our goal is always to provide modules that work with `include $module` and will result in a working service with sane defaults.

Reply:
Setting this to 1.0.1 means, that we must update puppet-hdm after every hdm release. Besides this: versions differ between docker and rvm setup.

Reply:
was thinking the same 🤔

Reply:
Version has been set to 1.0.1